src/mesh.rs

```diff
@@ -371,6 +381,25 @@ impl<'a> Primitive<'a> {
         self.json
     }
 
+    /// Returns the bounds (min/max) of the POSITION attribute if there is one, otherwise `None`.
+    /// May panic for invalid glTF files. Use json::validation::Validate::validate_minimally
+    /// to handle this gracefully.
+    pub fn bounds(&self) -> Option<Bounds> {
```

Comment:
I had a tuple initially, but changed to struct because you can't see which is `min` and which is `max` from the declaration alone when using a tuple.

Will change the name.